examples/u4a-deploy-tutorial.md

```diff
@@ -0,0 +1,245 @@
+## 1 Intall kubebb/core
+
+首先需要部署基本组件 `kubebb/core`
+
+```shell
+helm repo add kubebb https://kubebb.github.io/components
+kubectl create ns kubebb-system
+helm -nkubebb-system install kubebb/core
+
+NAME                     READY   STATUS    RESTARTS   AGE
+kubebb-755784d5f-k8jhh   1/1     Running   0          2m14s
+```
+
+## 1.1 create repository
+
+创建包含 `u4a-component` 相关组件的 Repository
+
+```yaml
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: Repository
+metadata:
+  name: kubebb
+  namespace: kubebb-system
+spec:
+  url: http://1.1.1.1:9999/def
+  pullStategy:
+    intervalSeconds: 120
+    retry: 5
+```
+
+## 2 cluster-comonent
+
+直接创建 `componentplan`, clustr-component 会部署 `ingress-nginx` 需要选择部署节点名称，默认是`kind-worker`
+
+**注意，这里我们的 namesapce 选择是 u4a-system, 目前不要选择其他的 namespace，适配还不完全**
+
+```yaml
+apiVersion: core.kubebb.k8s.com.cn/v1alpha1
+kind: ComponentPlan
+metadata:
+  name: cluster-compnent
+  namespace: u4a-system
+spec:
+  approved: true
+  name: cluster-component
+  version: 0.1.0
```

Comment:
use https://github.com/kubebb/core/blob/main/tests/kind-config-3nodes.yaml this kubrnetes cluster as the sample cluster. And use node `kubebb-core-worker` as the deployedHost for ingress